templates/s3-bucket.yaml

```diff
@@ -256,8 +255,6 @@ Resources:
           Value: 'Alfresco Content Services'
         - Key: Stack
           Value: !Ref 'AWS::StackName'
-        - Key: Name
```

Comment:
I removed it because the name will now be auto-generated so leaving it as it was would result in an incorrect tag value, furthermore, it's now consistent with all the other Lambda functions defined in the templates i.e. none of the others define a Name tag.